sr_hand_config/1066/rates/sensor_data_polling.yaml

```diff
@@ -0,0 +1,36 @@
+generic_sensor_data_update_rate: {
```

Comment:
Ah, this is something I didn't know about! Adding now.

Reply:
If otherwise all files are the same, then maybe it is better to have rates in common folder instead of zillion copies or one copy per hand type. Adding MST everythere also doesn't hurt as all sensors are defined in every file too just MSTs were not added.

Reply:
MST stuff added in 505bbee